### ISSUE TITLE
Tables in callouts with metadata parse as malformed.

### ISSUE THREAD

**Issue by sailKiteV** [NicoNekoru/obsidan-advanced-table-xt#25]
- [Overview](#overview)
  - [Information](#information)
    - [Error](#error)
    - [Debug](#debug)
    - [Steps to reproduce](#steps-to-reproduce)
  - [Examples](#examples)
    - [Demonstration](#gif1)
- [Postscript](#postscript)

— Replies —
  - 

---
# Overview <a href="#user-content-overview" id="overview"></a>
When using Sheets Extended with a table that is inside a callout, if that callout has metadata, the table parsing believes the table to be malformed.

### Error <a href="#user-content-error" id="error"></a>
An error is thrown at the beginning of the table:

>    `` Error: `Malformed table` ``

### Debug <a href="#user-content-debug" id="debug"></a>
Running Obsidian's debug command produces the following:

```
SYSTEM INFO:
	Obsidian version: v1.4.14
	Installer version: v1.4.13
	Operating system: Windows 10 Home 10.0.22621
	Login status: logged in
	Catalyst license: supporter
	Insider build toggle: on
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 2
	Plugins enabled: 2
		1: Obsidian42 - BRAT v0.7.0
		2: Sheets Extended v1.2.3
```

### Steps to reproduce <a href="#user-content-steps-to-reproduce" id="steps-to-reproduce"></a>

1. Create a table inside a callout.
2. Add metadata to the callout using the pipe character.
3. Enter Reading View or some other full-rendered context.

## Examples <a href="#user-content-examples" id="examples"></a>
i have attached two files related to testing in the Sandbox Vault to demonstrate the occurrence of the issue.
1) [copy of `Sheets Extended Testing.md`](https://github.com/NicoNekoru/obsidan-advanced-table-xt/files/12821176/temp.md)
2) [.gif demonstrating bug](https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/75286796/068ffe31-417f-464c-9cba-fa20cf60c1c1)

### Demonstration of bug <a href="#user-content-demonstration-of-bug" id="gif1"></a>
![.gif demonstrating bug](https://github.com/NicoNekoru/obsidan-advanced-table-xt/assets/75286796/068ffe31-417f-464c-9cba-fa20cf60c1c1)

---
## Postscript <a href="#user-content-postscript" id="postscript"></a>
Thank you for your time, and if you require any other information please don't hesitate to ask.

Regards,
-sailKite

**Comment by NicoNekoru** [comment]
Ah looks like there was a slight oversight in the regex fix for #22. I have a found a working regex and will push to next patch.

**Comment by NicoNekoru** [comment]
Fix landed in 1.2.4
